HerPortal.BusinessLogic/Models/TrackedCsvFile.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace HerPortal.BusinessLogic.Models;
+
+public class TrackedCsvFile
+{
+    public int Id { get; set; }
```

Comment:
This column doesn't have to be defined here if you don't use it for anything business related - you can define it in the DB context